web-server/src/utils/response/custom-error/types.ts

```diff
@@ -1,12 +1,43 @@
 export type ErrorResponse = {
   errorType: ErrorType;
   errorMessage: string;
-  errors: string[] | null;
   errorRaw: any;
-  errorsValidation: ErrorValidation[] | null;
+  errors: ErrorArray | null;
   stack?: string;
 };
 
 export type ErrorType = 'General' | 'Raw' | 'Validation' | 'Unauthorized';
 
-export type ErrorValidation = { [key: string]: string };
+export class ErrorArray {
+  private dict: { [key: string]: string[] };
+
+  constructor() {
+    this.dict = {};
+  }
+
+  put(key: string, value: string) {
+    if (key in this.dict) {
+      this.dict[key].push(value);
+    } else {
+      this.dict[key] = [value];
+    }
+  }
+
+  get(key: string) {
+    let value = this.dict[key];
+    value = value ? value : [];
+    return value;
+  }
+
+  get isEmpty() {
+    return Object.keys(this.dict).length === 0;
+  }
+
+  extend(other: ErrorArray) {
+    for (const key in other.dict) {
+      for (let i = 0; i < other.get(key).length; i++) {
+        this.put(key, other.get(key)[i]);
+      }
+    }
+  }
+}
```

Comment:
It might be better to explicitly return a:

```
type TaskCreateError = {
  id?: string;
  description?: string;
  ...
}
```
So that the client can get the return value, see if it's error 400, then pull out the error messages and dump them on to the right location on the page. A very forgiving dict doesn't help the client.